### PR TITLE
all: version 0.35.0

### PR DESCRIPTION
**Pull request by aykevl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+  - `rp2350`: add support for the new RP2350
+  - `wasm`: revise and simplify wasmtime argument handling
+  - `wasm`: support `//go:wasmexport` functions after a call to `time.Sleep`
+  - `wasm`: correctly return from run() in wasm_exec.js
+  - `wasm`: call process.exit() when go.run() returns
+  - `windows`: don't return, exit via exit(0) instead to flush stdout buffer
+* **boards**
+  - add support for the Tillitis TKey
+  - add support for the Raspberry Pi Pico2 (based on the RP2040)
+
```
Comment (deadprogram) [inline CHANGELOG.md:67]: Also needs to mention Pimoroni Tiny2350 here please :smile_cat:

**Comment by deadprogram** [comment]
@aykevl this PR needs to be updated with a few things that were merged after you wrote the changelog. Do you want me to do it?

**Comment by deadprogram** [comment]
Updated with the last minute changes.

**Comment by aykevl** [comment]
Thank you!

**Pull request merged**